src/librustdoc/clean/types/tests.rs

```diff
@@ -69,3 +69,14 @@ fn should_not_trim() {
     run_test("\t    line1  \n\t    line2", "line1  \nline2");
     run_test("    \tline1  \n    \tline2", "line1  \nline2");
 }
+
+#[test]
+fn is_same_generic() {
```

Comment:
This exists as a doctest too, right? Did you add it here as well in case the docs are changed?

Reply:
👍 Thanks for adding the clarifying comment to the "doctest".